example_notebooks/example_logs/cp_idrstream.log

```diff
@@ -0,0 +1,69 @@
+2022-12-16 12:58:27,312 - INFO - IDR stream initialized
```

Comment:
It would probably be best to delete any extra text here that wouldn't appear from a successful idrstream run. The idea here is that anyone that tries to replicate `idrstream_cp.ipynb` would be able to see an example perfect log file here (so they can compare their output). This same change should be made for `dp_idrstream.log`

Reply:
:+1: